src/ViewModels/src/DeploymentDialog/DeploymentDialogViewModel.cs

```diff
@@ -36,6 +36,8 @@ public class DeploymentDialogViewModel : AbstractViewModel, IDeploymentDialogVie
         private CloudFoundryInstance _selectedCf;
         private CloudFoundryOrganization _selectedOrg;
         private CloudFoundrySpace _selectedSpace;
+        private string manifestPathLabel;
```

Comment:
not sure what the naming convention is, however these names are inconsisten. Some have _ prefix and some not.

Reply:
Ah, yeah I think this is a product of using the "Convert to full property" feature in VS; it creates a private var to track the state of the prop you're converting but doesn't abide by our naming conventions apparently 😅 

Resolved in 66d1f23